salt/modules/opkg.py

```diff
@@ -106,21 +106,25 @@ def _get_restartcheck_result(errors):
     return rs_result
 
 
-def _process_restartcheck_result(rs_result):
+def _process_restartcheck_result(rs_result, **kwargs):
```

Comment:
Why `**kwargs` and not just `always_restart_services=True` as a keyword argument?